src/lib.rs

```diff
@@ -674,6 +674,15 @@ impl Client {
         self.by(search.into()).await
     }
 
+    /// Find an element on the page, starting from the given element.
+    pub async fn find_from(
```

Comment:
I wonder if we should _just_ have the methods on `Element`, and not these "nested find" methods on `Client`. That would also remove the need for the `clone` as you could operate entirely on `&mut Element`.

Reply:
Sounds like it would make sense; I'll try to do it that way!

Reply:
`Element::find` also makes more sense semantically, but I don't think we can really get rid of the clone, as a `WebDriverCommand::FindElementElement` requires an owned `WebElement`.